generators/feign-client/templates/src/test/java/_package_/client/AuthorizationHeaderUtilTest.java.ejs

```diff
@@ -82,7 +82,7 @@ class AuthorizationHeaderUtilTest {
 
     @Test
     void getAuthorizationHeader_Authentication() {
-        UsernamePasswordAuthenticationToken authenticationToken = new UsernamePasswordAuthenticationToken("principal", "credentials");
+        var authenticationToken = new UsernamePasswordAuthenticationToken("principal", "credentials");
```

Comment:
I'm not a big fan of var: most of the time, it obliterates the typesafety of this wonderful programming language which is not suffixed by script.

Do you want to introduce this change?

Reply:
If Sonarqube tells, we should do ^^. Better to follow the rules than hacking/deactivating them.

Ok to merge, thank you for the justification